### ISSUE TITLE
Metal Gear Solid Snafu

### ISSUE THREAD

**Issue by BobbyBoyGaming** [unexpectedpanda/retool#222]
## Dat
Redump.org
Sony - PlayStation - Datfile (10668) (2022-08-30 15-42-50).dat 

## Issue

If Retool is setup is left with English-speaking default game-region priority list, it will select Metal Gear solid Integral but then also select Metal gear solid VR missions USA Release.

+ Metal Gear Solid - Integral (Japan, Asia) (En,Ja) (Disc 1)
	- Metal Gear Solid (Europe) (Disc 1)
	- Metal Gear Solid (USA) (Disc 1)
	- Metal Gear Solid (USA) (Disc 1) (Rev 1)
+ Metal Gear Solid - Integral (Japan, Asia) (En,Ja) (Disc 2)
	- Metal Gear Solid (Europe) (Disc 2)
	- Metal Gear Solid (USA) (Disc 2)
	- Metal Gear Solid (USA) (Disc 2) (Rev 1)
+ Metal Gear Solid - VR Missions (USA)
	- Metal Gear Solid - Special Missions (Europe) (En,Fr,De,Es,It)

It would be nice if it could select a consistent region for a series of games. For example if its gonna pick the Integral Japanese release, then maybe it should select the Japanese VR mission disc too. 

It may be very laborious to do this for too many titles but at least for the high-profile ones like Metal gear solid it would be a good idea. 

In other words select a region for a game, but then have the option to keep the region for that title/series/collection consistent. 
I supposed it can be done through the inclusions exclusions option. but its kinda sneaky, because you cant know what the program will do until after you have scanned/repaired your entire rom set.

**Comment by unexpectedpanda** [comment]
There's no feature at this point in time to ensure a bundle of discs stay together. The logic involved there is complicated, and requires yet more data to be added to the clone list. Not saying never, just not now.

In this case, the selection is correct and intentional. The VR Missions equivalent in the Japanese/Asia release (_Metal Gear Solid - Integral (Japan, Asia) (Disc 3) (VR-Disc)_) is in [Japanese only](http://redump.org/disc/426/), and since you selected English, you got a version that supported the language you wanted.

**Comment by BobbyBoyGaming** [comment]
Oh okay, i did not know about the language thing for the VR Mission Disc 3, but I will have to test it out myself in the future.

**Comment by BobbyBoyGaming** [comment]
I just loaded the Integral VR-Disc and it is in English, the audio, the text, everything. Just an FYI in case you were curious

**Comment by unexpectedpanda** [comment]
Good to know. I can make tweaks on my side, but ultimately this needs to be fixed at Redump. I'll see if that's possible.

**Comment by unexpectedpanda** [comment]
Going deeper than the title screen:

![Metal Gear Solid - Integral (Japan, Asia) (Disc 3) (VR-Disc)-220926-211812](https://user-images.githubusercontent.com/15709984/192265319-9f0042fd-e455-471a-9e7a-06c9ed4d56ce.png)
![Metal Gear Solid - Integral (Japan, Asia) (Disc 3) (VR-Disc)-220926-211906](https://user-images.githubusercontent.com/15709984/192265345-0e917956-07d2-438f-bd32-0e5f1422d495.png)
![Metal Gear Solid - Integral (Japan, Asia) (Disc 3) (VR-Disc)-220926-212233](https://user-images.githubusercontent.com/15709984/192265354-2f7104da-aa5c-4249-816d-e08f10ea271f.png)
![Metal Gear Solid - Integral (Japan, Asia) (Disc 3) (VR-Disc)-220926-212424](https://user-images.githubusercontent.com/15709984/192265363-14fa90ce-08c5-44b9-ab9c-d2bd81e8fa54.png)

Looks like it's mostly playable, although in a number of sections Japanese subtitles are permanently on the screen. Unfortunately the movies in "Extra" are entirely in Japanese, and there are definitely prompts in the game that are solely in Japanese. It's more a weird hybrid than a properly English-supported title.

This prompted me to try the main game. The spoken dialogue is all in English, however the player has a choice between Japanese and English subtitles:

![Metal Gear Solid - Integral (Japan, Asia) (En,Ja) (Disc 1)-220926-215210](https://user-images.githubusercontent.com/15709984/192270802-e72c7582-e01e-406b-afe0-3147e9162375.png)

Unfortunately the main game is also a bit of a hybrid, with text that's only in Japanese in some places:

![Metal Gear Solid - Integral (Japan, Asia) (En,Ja) (Disc 1)-220926-215219](https://user-images.githubusercontent.com/15709984/192270603-fd97c45a-3dfe-4b24-b727-a5568142bc68.png)

![Metal Gear Solid - Integral (Japan, Asia) (En,Ja) (Disc 1)-220926-215305](https://user-images.githubusercontent.com/15709984/192270639-5dd50b01-6834-4a5d-acde-b70c018fed48.png)

![Metal Gear Solid - Integral (Japan, Asia) (En,Ja) (Disc 1)-220926-215640](https://user-images.githubusercontent.com/15709984/192270668-f84b4a05-a976-4117-aa59-263726b6f903.png)

At this rate I'm thinking I should no longer promote it as an English-capable edition, and leave it for those who set Japan high in their preferences. Although then that has to compete with the original Japanese edition, which has spoken Japanese instead of English...

Oh the joys and complexities of video games.

**Comment by Keiyuu** [comment]
Came here to open an issue but noticed someone already mentioned it.

Integral should definitely not replace the US version when selecting only English! I just did that and it threw out the US version in favor of Integral. Yes, it has more features and it may be "playable" but there is still lots of Japanese text, including important things like in-game descriptions.

This is definitely not the version to pick when someone isn't familiar with the game and wants to play it for the first time in English.

**Comment by unexpectedpanda** [comment]
Apologies, I got caught up in trying to push v2 out, which has been taking a majority of my time. This has been corrected.
